examples/visualization/plot_image_gallery.py

```diff
@@ -23,18 +22,10 @@
     shuffle_files=True,
 )
 
-
-def unpackage_tfds_inputs(inputs):
-    return inputs["image"]
-
-
-train_ds = train_ds.map(unpackage_tfds_inputs)
-train_ds = train_ds.apply(tf.data.experimental.dense_to_ragged_batch(16))
+train_ds = train_ds.ragged_batch(16)
 
 keras_cv.visualization.plot_image_gallery(
-    next(iter(train_ds.take(1))),
+    train_ds,
```

Comment:
For the example, let's show both ways of calling this (maybe one with a Numpy array of images, and the other with a tf.data.Dataset

Reply:
Done